contracts/Controller.sol

```diff
@@ -780,16 +857,22 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
             }
         }
 
+        // get otoken details
```

Comment:
I don't think we need comments on every line of the code. The code itself should be self-explaining, comments like this `get otoken details` while the function name is called `getOtokenDetails` is just redundant IMO.

Reply:
unless the comments explain more than what the code says ;)